rules/awsrules/aws_s3_bucket_invalid_region.go

```diff
@@ -23,16 +21,27 @@ func NewAwsS3BucketInvalidRegionRule() *AwsS3BucketInvalidRegionRule {
 		attributeName: "region",
 		enum: []string{
 			"EU",
```

Comment:
If you use EU, terraform will fail when reading the resource, but the creation will succeed. Although it should not be used newly, EU may already exist as a valid value.
This means to allow false negatives, but it makes sense here.